app/javascript/shared/helpers/AudioNotificationHelper.js

```diff
@@ -7,3 +7,31 @@ export const playNotificationAudio = () => {
     console.log(error);
   }
 };
+
+export const playAudioFromIframe = () => {
+  try {
+    const audioEl = document.querySelector('#notif-audio');
+    audioEl.currentTime = 0;
+    audioEl.play();
+    audioEl.muted = false;
+
+    setTimeout(() => {
+      audioEl.muted = true;
+      audioEl.pause();
+    }, 300);
+  } catch (error) {
+    console.log(error);
+  }
+};
+
+export const newMessageNotification = data => {
+  if (document.hidden) {
+    playAudioFromIframe();
+  } else {
+    const { conversation_id: currentPageId } = window.WOOT.$route.params;
+    const { conversation_id: incomingConvId } = data;
+    if (currentPageId !== incomingConvId) {
+      playAudioFromIframe();
+    }
```

Comment:
Do we need to do this? I guess we should only alert the user with the sound only if the window is out of focus? 

@sojan-official thoughts?

Reply:
The ideal case is if there is no user interaction on the page for x minutes. I could be away from my desk to get my coffee and definitely the sound would help me not to miss out the conversation.

If we have the UI for disabling audio alert I would go for sound on all notifications even when the page is on focus

- Accounts with high volume conversations can have the sounds disabled.
- Accounts where volume is low, the notification would help me not to miss out on messages. 

We could take it as a second iteration to disable the notifications if the user is active on the page